<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditObjectDialog2</class>
 <widget class="QDialog" name="EditObjectDialog2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Object Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>500</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>531</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="DoubleInputForm" name="posXSlider" native="true"/>
    </item>
    <item>
     <widget class="DoubleInputForm" name="posYSlider" native="true"/>
    </item>
    <item>
     <widget class="DoubleInputForm" name="posZSlider" native="true"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="setPosXY">
        <property name="text">
         <string>Pos X=Y</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="setPosXZ">
        <property name="text">
         <string>Pos X=Z</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="setPosYX">
        <property name="text">
         <string>Pos Y=X</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="setPosYZ">
        <property name="text">
         <string>Pos Y=Z</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="setPosZX">
        <property name="text">
         <string>Pos Z=X</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="setPosZY">
        <property name="text">
         <string>Pos Z=Y</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="DoubleInputForm" name="scaleXSlider" native="true"/>
    </item>
    <item>
     <widget class="DoubleInputForm" name="scaleYSlider" native="true"/>
    </item>
    <item>
     <widget class="DoubleInputForm" name="scaleZSlider" native="true"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="setScaleXY">
        <property name="text">
         <string>Scale X=Y</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="setScaleXZ">
        <property name="text">
         <string>Scale X=Z</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="setScaleYX">
        <property name="text">
         <string>Scale Y=X</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="setScaleYZ">
        <property name="text">
         <string>Scale Y=Z</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="setScaleZX">
        <property name="text">
         <string>Scale Z=X</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="setScaleZY">
        <property name="text">
         <string>Scale Z=Y</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Degrees</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBoxRotX">
          <property name="value">
           <double>15.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QPushButton" name="rotateX">
          <property name="text">
           <string>Rotate + X-axis</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="rotNegX">
          <property name="text">
           <string>Rotate - X-axis</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Degrees</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBoxRotY">
          <property name="value">
           <double>15.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QPushButton" name="rotateY">
          <property name="text">
           <string>Rotate + Y-axis</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="rotNegY">
          <property name="text">
           <string>Rotate - Y-axis</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Degrees</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBoxRotZ">
          <property name="value">
           <double>15.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QPushButton" name="rotateZ">
          <property name="text">
           <string>Rotate + Z-axis</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="rotNegZ">
          <property name="text">
           <string>Rotate - Z-axis</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DoubleInputForm</class>
   <extends>QWidget</extends>
   <header>doubleinputform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditObjectDialog2</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditObjectDialog2</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
